### DISCUSSION TITLE
After released Hashlips v1.1.2 - Update

### DISCUSSION THREAD

**Discussion by hirushanme** [nftchef/art-engine#25] [Help + Q&A]
As you mention description this code builds with Hashlips generator (v.1.0.6 ).But now realsed Hashlips v1.1.2 with fixed bugs.can you Update this Code sir. Thanks you

**Comment by nftchef** [comment]
This codebase implements many of the same features in slightly different ways and maintains the majority of feature parody. I don't have any plans in the near future to merge the code. 

If there are any features missing, please let me know and I will do my best to add them
